diff --git a/test/Makefile.am b/test/Makefile.am
index b37d4cd..12b6056 100644
--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -1,15 +1,11 @@
 AUTOMAKE_OPTIONS=foreign
 
 TESTS =						\
-  t.chmod_dev					\
   t.echoarg					\
   t.falsereturn					\
-  t.mknod					\
   t.no_ld_preload				\
   t.no_ld_preload_link				\
   t.option					\
-  t.tar						\
-  t.touchinstall				\
   t.truereturn \
   t.xattr
 
diff --git a/test/Makefile.in b/test/Makefile.in
index f565485..5727ecf 100644
--- a/test/Makefile.in
+++ b/test/Makefile.in
@@ -459,15 +459,11 @@ top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 AUTOMAKE_OPTIONS = foreign
 TESTS = \
-  t.chmod_dev					\
   t.echoarg					\
   t.falsereturn					\
-  t.mknod					\
   t.no_ld_preload				\
   t.no_ld_preload_link				\
   t.option					\
-  t.tar						\
-  t.touchinstall				\
   t.truereturn \
   t.xattr
 
